CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [0.3.3] - 2022-05-31
```

Comment:
I think the numbering in the changelog is somehow wrong because this section should be 0.3.2 but there's already a section named 0.3.2 :thinking: